test/validation/api/crypto/odp_crypto_test_inp.c

```diff
@@ -270,6 +273,11 @@ static int alg_packet_op_enq(odp_packet_t pkt,
 		return rc;
 	}
 
+	if (!result.ok)
+		CU_ASSERT(odp_packet_has_error(pkt))
+	else
+		CU_ASSERT(!odp_packet_has_error(pkt));
+
```

Comment:
`CU_ASSERT()` needs to be enclosed in braces when used in an `if/else` statement. However a simpler test is just:
```
CU_ASSERT(result.ok != odp_packet_has_error(pkt));
```

Reply:
ok.